### ISSUE TITLE
Imported Widgets new id

### ISSUE THREAD

**Issue by miljan-aleksic** [stevengliebe/widget-importer-exporter#6]
Hi, thanks for this cool plugin, it helps a lot :)

I am hooking into the import process and what I am missing is knowing the new widget ID once imported. I would like to propose a hook which would look something like this:

```
do_action( 'wie_widget_imported', $widget, $new_instance_id_number );
```

Let me know your thoughts about.

**Comment by stevengliebe** [comment]
Hello, I'm glad you find it useful.

What I'm thinking is something like this. Since sidebars are looped then widgets and because there is a lot of data that could be made available and I'm not sure what all people might possibly need, I figure using an array would be flexible enough. What do you think?

Want to try this and see if it works for your need? If it does, I'll stick it in the next release. You can see I inserted the action right before the success message. I appreciate a second set of eyes on this.

```php
// Assign widget instance to sidebar
$sidebars_widgets = get_option( 'sidebars_widgets' ); // which sidebars have which widgets, get fresh every time
$new_instance_id = $id_base . '-' . $new_instance_id_number; // use ID number from new widget instance
$sidebars_widgets[$use_sidebar_id][] = $new_instance_id; // add new instance to sidebar
update_option( 'sidebars_widgets', $sidebars_widgets ); // save the amended data

// After widget import action
$after_widget_import = array(
	'sidebar_id'				=> $sidebar_id,
	'use_sidebar_id'			=> $use_sidebar_id,
	'widget_instance_id'		=> $widget_instance_id,
	'widget'					=> $widget,
	'id_base'					=> $id_base,
	'instance_id_number'		=> $instance_id_number,
	'new_instance_id_number'	=> $new_instance_id_number,
	'new_instance_id'			=> $new_instance_id,
);
do_action( 'wie_after_widget_import', $after_widget_import );

// Success message
if ( $sidebar_available ) {
	$widget_message_type = 'success';
	$widget_message = __( 'Imported', 'widget-importer-exporter' );
} else {
	$widget_message_type = 'warning';
	$widget_message = __( 'Imported to Inactive', 'widget-importer-exporter' );
}
```

**Comment by miljan-aleksic** [comment]
The action works perfectly, the only confusing part is the variable naming. My suggestion would be:

```
'widget' // []
'widget_type' // 'text'
'sidebar' // 'search'
'sidebar_old' // 'search'
'widget_id' // 'text-1'
'widget_id_old' // 'text-44'
'widget_id_num' // 1
'widget_id_num_old' // 44
```

**Comment by stevengliebe** [comment]
That would certainly make it easier to use.

Do you want to submit a pull request with the change as you described?

**Comment by stevengliebe** [comment]
Just released 1.2.1 with this. Thank you for contributing.
